01-node-tutorial/answers/9-path-module.js

```diff
@@ -0,0 +1,12 @@
+const path = require('path')
+
+console.log(path.sep)
+
+const filePath = path.join('\content', 'subfolder', 'test.txt')
```

Comment:
```suggestion
const filePath = path.join('/content', 'subfolder', 'test.txt')
```
Close, this just needs to use a forward slash instead of a back slash. A forward slash marks the start of an absolute path.